### ISSUE TITLE
Two Snipe Accounts - One finds snipable pokemon, other one doesnt. 

### ISSUE THREAD

**Issue by Frankkz** [NECROBOTIO/NecroBot#4189]
Running 2 accounts, same config.
Setup PogoLocationFeeder to snipe pokemon.

One account finds the pokemon, the other one doesnt.
SnipingScanOffset: 0.003

Account 1:
[01:37:35] Sniping a Aerodactyl with 100 IV at 47.599887,-122.328804...
[01:37:38] Used Razz Berry | 12 remaining
[01:37:40] (CatchSuccess) | (Normal) Aerodactyl Lvl: 14 CP: (866/866) IV: 100.00% | Chance: 16% | 3867214.15m dist | with a UltraBall (0 left). | 210 EXP earned | Candies: 91 | lat: 47.599887 long: -122.328805
Account 2:
[01:36:48] Sniping a Aerodactyl with 100 IV at 47,599887,-122,328804...
[01:36:50] Did not find a Pokemon within the SnipingScanOffset!

**Comment by djdookie** [comment]
This shouldn't happen. Are you sure you used different credentials in auth.json?

**Comment by Frankkz** [comment]
Yes I'm sure; Picture of the 2 bots running: http://image.prntscr.com/image/53a26455cb3840029705cbed4743ce97.png

**Comment by Frankkz** [comment]
Just watched the same thing happen again. This time on version 0.8.5. One account found the vaporeon, the other 2 didn't. They are all using the same configuration.
http://image.prntscr.com/image/18e5caa3ad5045f586f62d23443fd040.png

**Comment by dimonstrer** [comment]
Same problem

**Comment by Frankkz** [comment]
Again(Alakazam): http://image.prntscr.com/image/25e1028b484546b3813a422c739d4ea2.png
and again(Electabuzz): http://image.prntscr.com/image/97986e1aaee54b0fb0644db8838462a2.png

**Comment by Poupoul80** [comment]
I dont understand how to enabled snipe. somebody can help me please? thanks a lot. :)
